### DISCUSSION TITLE
Missing properties in useQuery

### DISCUSSION THREAD

**Discussion by nathanWanderTech** [urql-graphql/urql#1350] [Q&A] [answered]
Hi, 

I'm using "urql 1.11.6" and got this weird issue in using useQuery(). Here is my code:

`const [{ data, fetching }] = useQuery({
    query: PROGRAM_getPrograms,
    variables: {
      page: 1,
      search_name: "",
      order_by: "NUMBER_OF_THINGS",
      other: "",
      direction:  SortDirection.ASC,
    },
  });`

useQuery() did not send all properties but the request payload only has 3 following properties
`{
      page: 1,
      search_name: "",
      direction:  SortDirection.ASC,
    }`

What is the right understanding for this issue and how to fix it?

**Comment by JoviDeCroock** [comment, marked as answer]
Could you show us  what `PROGRAM_getPrograms` looks like and your client itself, what you are saying is that `other` and `order_by` are missing in the actual fetch call to your server right?
